cogs/Logs.py

```diff
@@ -82,6 +85,20 @@ async def on_user_update(self, before, after):
 
     @Cog.listener()
     async def on_member_update(self, before, after):
+        before_roles = []
+        after_roles = []
+        for role in before.roles:
+            before_roles.append(str(role.mention))
+
+        before_roles.reverse()
+        before_roles.pop()
+
```

Comment:
messssssssssssssssssssssyyyyyyyyy

Reply:
the whole section

Reply:
that's true, but as I said I didn't code it and also don't have time to refactor code

Reply:
otherwise I would just do a few simple list comprehensions and clean it up, but that all needs time to code and test afterwards